app/scripts/views/mixins/resend-mixin.js

```diff
@@ -8,12 +8,17 @@
 'use strict';
 
 define([
-], function () {
+  'models/email-resend'
+], function (EmailResend) {
   var SHOW_RESEND_IN_MS = 5 * 60 * 1000; // 5 minutes.
-  var TOO_MANY_ATTEMPTS = 4;
 
   return {
-    _attemptedSubmits: 0,
+
+    initialize: function () {
+      this._emailResend = new EmailResend({
```

Comment:
See https://github.com/mozilla/fxa-content-server/pull/2553/files#r32203356.

Instead of passing in `onTriesExceeded`, attach the handler using 
```js
this._emailResend.on('maxTriesExceeded', ...
```